templates/learn.md

```diff
@@ -12,12 +12,13 @@ Some have Lean 3 versions, but there is no point learning Lean 3 at this stage.
 ## Hands-on approaches
 
 * Whatever your background, if you want to dive right away, you can play the
-  [Natural Number Game](https://adam.math.hhu.de/#/g/hhu-adam/NNG4). 
+  [Natural Number Game](https://adam.math.hhu.de/#/g/hhu-adam/NNG4).
   This is an online interactive Lean tutorial
   focused on proving properties of the elementary operations on natural numbers.
-  Similar games for [Logic and Set Theory](https://adam.math.hhu.de/#/) are available.
+  The [Lean Game Server](https://adam.math.hhu.de/#/) hosts various learning games including
+  [Set Theory, Logic, and Robo (a story about undergrad mathematics)](https://adam.math.hhu.de/#/).
```

Comment:
```suggestion
  Set Theory, Logic, and Robo (a story about undergrad mathematics).
```
This is the same link as "Lean Game Server." Let's either link directly to the individual games, or not use a link here.

Reply:
sounds good! done!